### ISSUE TITLE
Sprint 17 Goal

### ISSUE THREAD

**Issue by hmitsch** [mozilla-iam/iam-project-backlog#211]
## Prioritized Themes
* Mozillians.org account verification for Google and Github identities (incremental rollout)
* Moving forward, Mozillians.org passwordless login will support the use of only one, identity verified email address, per user (incremental rollout)
* Have a first property using IAM Group Management to control access
* Unblock the expansion of the Q2 OKR

## Mozillians.org
- [x] #212 Mozillians.org to support identity upgrades and single identity ONLY for access management groups. (Merged to Mozilla/mozillians

## SSO-Dashboard
- [x] Add MDN Icons and login.mozilla.com to production dashboard
- [x] Alert UI Color Updates #231 

## CIS
- [x] https://github.com/mozilla/parsys/issues/236 Allow Jenkins in Mozillians infra to be callable and return status to cis code pipeline (wontfix) Jenkins plugin not suited for cross-account roles.
- [ ] #202 Complementary issue to the above.  Set hooks in code pipeline
- [x] #222 Define and document automatic expiration of access 
- [x] #174 Design CIS "Profile" API
- [x] #223 work-around to integrate LDAP groups in Auth0 instead of CIS until LDAP publisher is live 100% In production
- [x] #212 Implement mozillians.org identity upgrades. Merged to Mozilla/mozillians

## Federated access proxy
- [ ] #225 POC STS token support
- [x] #226 Deploy Forbidden Page in Production 100%

## OKR Q401
- [ ] #213 Account Verification ( Agreement on architecture and begin tasks ) 33%
- [ ] #115 LDAP Subscriber ( Agreement on architecture and begin ) 50%

## Infrastructure
- [x] [Bug 1359985](https://bugzilla.mozilla.org/show_bug.cgi?id=1359985) Auth0 for jenkins.services.mozilla.community authentication
- [x] #230 Deploy a new instance of Mozillians for CIS testing.  CIS_Staging => Mozillians CIS Test => Auth0 Dev
- [ ]  #171 Migrate RPs to OIDC conformant mode
- [ ] Migrate LDAP from SCL3 to MDC1/2

## ParSys Mental Sanity
- [ ] Mozillians Infrastructure: Update [Mozillians runbook](https://wiki.mozilla.org/Websites/Mozillians.org) - via [Bug 1404630](https://bugzilla.mozilla.org/show_bug.cgi?id=1404630) 
Document and figure out Participation system's escalation policy and onwed services
- [ ] ReMo: [Bug 1404725](https://bugzilla.mozilla.org/show_bug.cgi?id=1404725) Multiple entries found by clicking on a marker
- [x] ReMo: [Bug 1405484](https://bugzilla.mozilla.org/show_bug.cgi?id=1405484) Missing voting results email
- [ ] ReMo: [Bug 1405939](https://bugzilla.mozilla.org/show_bug.cgi?id=1405939) Rep website keep logged me out
- [x] Mozillians: https://github.com/mozilla/parsys/issues/237 Add external accounts in the user export for mozillians.org

## Good for the soul items
* Yousef and Andrew to begin discussion PoC on Kubernetes in AWS + RBAC to support Portland BSides and community ops.
